Solutions/InsiderRiskManagement/Analytic Rules/InsiderRiskyAccessByApplication.yaml

```diff
@@ -0,0 +1,35 @@
+id: 15386bba-dc70-463f-a09f-d392e7731c63
+name: (Private Preview) Insider Risk - Risky User Access By Application
+description: |
+  'Microsoft 365 Insider Risk Management Triggered An Alert'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - SigninLogs 
+queryFrequency: 6h
+queryPeriod: 6h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
+relevantTechniques:
+  - T1204
+query: |
+  SigninLogs
+  | where RiskState == "atRisk"
+  | project UserPrincipalName, Location, AppDisplayName, RiskState
+  | evaluate basket(0.01) // Adjust & Tune Thresholds within Organzational Requirements
+  // | where Percent > 50 // Adjust & Tune Thresholds within Organzational Requirements
+  | where UserPrincipalName <> ""
+  | where AppDisplayName <> ""
+  | project Percent, UserPrincipalName, Location, AppDisplayName, RiskState
+  | sort by Percent desc
+  | limit 250
```

Comment:
With evaluate, I doubt you would get 250, but same question as the previous query.  Can we mostly prevent high number of results?